lib/results.js

```diff
@@ -1,5 +1,5 @@
 var defined = require('defined');
-var EventEmitter = require('events').EventEmitter;
+var EventEmitter = require('events'); EventEmitter = EventEmitter.EventEmitter || EventEmitter;
```

Comment:
i think two lines is better :-)

however, since `require('events') === require('events').EventEmitter` is true in node 0.12 and later, and we need to maintain compat back to 0.8, it seems like the better choice is to leave this line as it is.